ramls/examples/hridsettings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "ca81eafd-ddb8-416a-89e1-4f772ab6afee",
+  "instance": {
+    "prefix": "in",
+    "startNumber": 1
+  },
+  "holding": {
```

Comment:
Hmm, is a holding singular, for the records themselves we refer to an individual record as a holdings record? 

I’m aware this is rather pedantic. The UI headings for these settings appears to be inconsistent, in that item is singular, and instances and holdings are plural, which adds to my confusion. 

I’m not really sure whether to try to raise rugs?

Reply:
Oh, my bad. I didn't look carefully at the UI settings page mockups to try to correlate the property names to the section headers, though they are quite verbose. I don't have a problem changing `holding` to `holdingsRecord`. Do you think that makes these intended use for these settings clearer? I could also make them all plural if that is better.

I agree that the UI itself is inconsistent and likely needs to be addressed. Perhaps they should have used "Inventory instance and MARC bibliographic records" instead of "Inventory instances and MARC bibliographic records". I'm not sure how to break down the "and" here. Is it "Inventory instances" and "MARC bibliographic records" or is it "Inventory instances records" and "MARC bibliographic records"? Given there is an "Inventory item records", I think they mean "Inventory instances records".

Reply:
I don't really mind if these use the singular or plural of the different records, as long as it is consistent. What I don't know is what the singular is for holdings (as each record represents a set, so the singular could be holdings).

I think `holdings` vs `holdingsRecord` depends upon the plurality question and whether we general refer to this as holdings record. Most of the time, when folks refer to an item, they don't say item record, rather item. I wonder if `record` in the context of holdings is a domain term rather than a technical term.

Reply:
I think the and is somewhat misleading since we adapted the requirements.

It used to be that external parts of FOLIO could get an instance HRID and assign it to a MARCCat record or a source record before the instance existed. We've since changed it to only allocate HRIDs upon inventory record creation.

I think this means that these settings don't actually affect MARC records, they are only mapped to them. I'm not sure if I know how or whether I should try to express that.

Reply:
> Singular is holdings or holdings record. Plural is holdings or holdings records.

I think we can pick whether we have plural or singular property names. It appears that either way, the holdings entry should be `holdings`.

I don't mind which we pick, as long as we are consistent. Given the naming for holdings, it might be that folks less familiar with the domain might find the plural less confusing.

Reply:
I am going to pluralize each field by adding an 's'. So, "instances", "holdings" and "items".